lib/Page.js

```diff
@@ -62,10 +63,43 @@ function calculateNewBaseUrl(filePath, root, lookUp) {
   return calculate(filePath, []);
 }
 
+function insertFlexDiv(pageData) {
+  const $ = cheerio.load(pageData);
+  const footers = $('footer');
+
+  footers.each((index, footer) => {
+    if (!($(footer).attr('id') === 'flexDiv')) {
+      $(footer).before(FLEX_DIV_HTML);
+    }
+  });
+  return new Promise((resolve) => {
+    resolve($.html());
+  });
+}
+
 function unique(array) {
   return array.filter((item, pos, self) => self.indexOf(item) === pos);
 }
 
+function removeDuplicateFooter(pageData) {
```

Comment:
`removeDuplicateFooters` (end with 's' because the method is capable of removing more than one footer).